src/components/sidebar/sidebar.scss

```diff
@@ -1,102 +1,87 @@
 @use '@carbon/colors';
 @use '@carbon/type';
 
-.sidebar {
-  width: 12rem;
-  min-height: 8rem;
-  overscroll-behavior: contain;
-  margin-right: 1rem;
-}
-
-.sidebarList {
-  max-height: 100%;
-}
-
-.sidenavActions {
-  margin-left: 0.6rem;
-}
-
-@media all and (device-width: 600px) and (device-height: 1024px) and (orientation: portrait) {
-  .sidebar {
-    width: 11rem;
-    max-height: 500px;
-    margin-right: 20px;
-    position: fixed;
-  }
-
-  .sidebarList {
-    max-height: 200px;
-    overflow-y: scroll;
-  }
-}
-
-.link {
-  margin: 0.375rem 0 0.375rem 0.5rem;
-  font-size: 1rem;
-  line-height: 1.43;
-  letter-spacing: 0.16px;
-  color: colors.$gray-100;
-  cursor: pointer;
-
-  :hover {
-    outline: none;
-  }
-}
-
-.section {
+.tab {
```

Comment:
I think `.tab` is a pretty bad name here. Surely "page" or "pageLink" is better?